src/api.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Usage metrics JSON API. Simple web.py based API to a
+PostgreSQL database that runs on port 8080.
+
+usage: python metrics.py
+
+(c) Javier Arias, Open Book Publishers, May 2019
+Use of this software is governed by the terms of the MIT license
+
+Dependencies:
+  python-dateutil==2.4.2
+  psycopg2==2.6.1
+  web.py==0.38
+"""
+
+import re
+import os
+import web
+import json
+import jwt
+import datetime
+from aux import logger_instance, debug_mode
+from errors import (Error, internal_error, not_found, NORESULT, UNAUTHORIZED,
+                    FORBIDDEN, BADFILTERS, BADPARAMS)
+
+# get logging interface
+logger = logger_instance(__name__)
+web.config.debug = debug_mode()
+# You may disable JWT auth. when implementing the API in a local network
+JWT_DISABLED = False
+# Get secret key to check JWT
+SECRET_KEY = ""
+try:
+    if 'JWT_DISABLED' in os.environ:
```

Comment:
suggestion: `os.getenv()`  can make this a bit cleaner.

Reply:
I've replaced the `if 'SECRET_KEY'`, but I've kept `if 'JWT_DISABLED'` as we want to keep the parameter optional and default to false